project_agile_analytic/views/project_agile_analytic_views.xml

```diff
@@ -0,0 +1,194 @@
+                <field name="type_id"/>
+                <field name="task_id"/>
+                <field name="stage_id"/>
+                <field name="user_id"/>
+                <group string="Group By..." expand="0" name="groupby">
+                    <filter string="Project" context="{'group_by':'project_id'}" name="group_project_id"/>
+                    <filter string="Type" context="{'group_by':'type_id'}" name="group_type_id"/>
+                    <filter string="Task" context="{'group_by':'task_id'}" name="group_task_id"/>
+                    <filter string="Stage" context="{'group_by':'stage_id'}" name="group_stage_id"/>
+                    <filter string="Assignee" context="{'group_by':'user_id'}" name="group_user_id"/>
+                </group>
+            </search>
+        </field>
+    </record>
+    <record id="view_project_agile_analytic_line_graph" model="ir.ui.view">
+        <field name="name">project.agile.analytic.line.view.graph</field>
+        <field name="model">project.agile.analytic.line.view</field>
+        <field name="arch" type="xml">
+            <graph string="Analytic Entries">
+                <field name="project_id" type="row"/>
+                <field name="type_id" type="row"/>
+                <field name="stage_id" type="row"/>
+                <field name="task_id" type="row"/>
+                <field name="user_id" type="col"/>
+                <field name="duration" type="measure"/>
+            </graph>
+        </field>
+    </record>
+    <record id="view_project_agile_analytic_line_pivot" model="ir.ui.view">
+        <field name="name">project.agile.analytic.line.view.pivot</field>
+        <field name="model">project.agile.analytic.line.view</field>
+        <field name="arch" type="xml">
+            <pivot string="Analytic Entries" >
+                <field name="project_id" type="row"/>
+                <field name="type_id" type="row"/>
+                <field name="stage_id" type="row"/>
+                <field name="task_id" type="row"/>
+                <field name="user_id" type="col"/>
+                <field name="duration" type="measure"/>
+            </pivot>
+        </field>
+    </record>
+
+    <menuitem
+        name="Analytic Entries"
+        parent="project.menu_project_report"
+        action="project_agile_analytic_line_action_entries"
+        id="menu_task_analytic_lines_tree"
+        sequence="35"
+        groups="project.group_project_manager"
+    />
+
+
+    <record id="view_task_form2_analytics" model="ir.ui.view">
+        <field name="name">view_task_form2_analytics</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.view_task_form2"/>
+        <field name="arch" type="xml">
+            <div name="button_box">
+                <field name="analytic_enabled" invisible="1"/>
+                <button name="%(project_agile_task_analytic_line_action)d" type="action" groups="project.group_project_manager"
```

Comment:
CAn you improve the readability of the XML line?

Reply:
Hope it's ok now ...